tests/tests_change_settings.yml

```diff
@@ -0,0 +1,77 @@
+- hosts: all
+  tasks:
+    - name: Ensure required packages are installed
+      package:
+        name: tuned
+        state: present
+
+    - name: Ensure kernel settings profile directory exists
+      file:
+        path: /etc/tuned/kernel_settings
+        state: directory
+        mode: 0755
+
+    - name: Generate a configuration for kernel settings
+      copy:
+        src: tuned/etc/tuned/basic_settings/tuned.conf
+        dest: /etc/tuned/kernel_settings/tuned.conf
+        mode: 0644
+
+    - name: Ensure required services are enabled and started
+      service:
+        name: tuned
+        state: restarted
+        enabled: yes
+
+    - name: apply kernel_settings
+      include_role:
+        name: linux-system-roles.kernel_settings
+      vars:
+        kernel_settings_sysctl:
+          - name: fs.file-max
+            value: 400000
+          - name: kernel.threads-max
+            state: absent
+        kernel_settings_sysfs:
+          - name: /sys/kernel/debug/x86/nmi_longest_ns
+            value: 2000000
+        kernel_settings_bootloader_cmdline:
+          - name: spectre_v2
+            value: "on"
+          - name: quiet
+            state: absent
+
+    - name: reboot the machine - see if settings persist after reboot
+      reboot:
+        test_command: tuned-adm active
+
+    - name: check after reboot for ns_longest
+      shell: grep 2000000 /sys/kernel/debug/x86/nmi_longest_ns
```

Comment:
I think `tuned-adm verify` checks to see if *all* of the settings in the profile are correctly applied to the system. @yarda @olysonek please confirm

Reply:
Also note - use `command` instead of `shell` unless you need to use bash logic, piping, or variable constructs

Reply:
Ok. I switched to command everywhere but I need pipes. Ok. Need to review again and change.

Reply:
> Ok. I switched to command everywhere but I need pipes. Ok. Need to review again and change.

I don't think you need pipes or redirection here.  But I suppose since this is test code and not run-time code, we can relax some of the restrictions against using `shell` and other unsafe constructs, in the interest of making it easier for test writers.

Reply:
Hello. Thanks. So what do you propose to use to check runtime values?